dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
+ * (c) 2015 Vokal LLC
+ * https://github.com/vokal/angular-svg-radial
+ */
+angular.module("svgRadial", []).directive("radialProgress", function() {
+    "use strict";
+    return {
+        restrict: "E",
+        transclude: true,
+        replace: true,
+        template: '<div class="radial-progress">' + '<div class="base"></div>' + '<div class="radial">' + '<div class="piece"></div>' + '<div class="piece"></div>' + "</div>" + '<div class="transclude" ng-transclude></div>' + "</div>",
```

Comment:
So all beautify is really doing is pulling out  700 bytes of whitespace.  If this is mainly just to assist a developer who wants to see non-minified code, it's just as easy for them to refer to the source file instead of the minified one.  This third file feels very superfluous.

Should move the source file to `/source`, too.